docs/requirements.md

```diff
@@ -0,0 +1,146 @@
+## Hydrophonitor CLI
+
+### Requirements
+
+Introduces a command line utility called `hp-cli` which interfaces with a Hydrophonitor, a Raspberry Pi based hydrophone passive acoustic monitoring system.
+
+### Supported commands
+
+#### 1. Import
+
+Introduces the following command:
+
+```sh
+hp-cli import DEVICE --output-path OUTPUT [options]
+```
+
+```
+Required arguments:
+
+DEVICE                      Path to USB mass storage or SD card where data will
+                            be imported from.
+
+--output-path OUTPUT        Path to where the directory for imported data
+                            will be created and data will be imported.
+
+Supported options:
+
+--merge-wavs                Merge all wav files in one deployment into one wav file.
+                            Resulting file will have as its name <timestamp>_audio.wav
+                            where the timestamp is the timestamp of the first wav file.
+
+--no-metadata               Do not prompt the user for submitting metadata about a deployment.
```

Comment:
Maybe instead of this we would have metadata fields as options which would then forego the prompt.

I was also thinking about the categorization method. Maybe tags would be the best? You could put `--tags ["andenes", "norway", "whale2sea"]` for example.

```
--name
--tags
--team
```